Mathlib/Algebra/Module/ZLattice/Basic.lean

```diff
@@ -607,4 +614,79 @@ instance instCountable_of_discrete_submodule {E : Type*} [NormedAddCommGroup E]
   simp_rw [← (Module.Free.chooseBasis ℤ L).ofZLatticeBasis_span ℝ]
   infer_instance
 
+end NormedLinearOrderedField
+
+section comap
+
+variable (K : Type*) [NormedField K] {E F : Type*} [NormedAddCommGroup E] [NormedSpace K E]
+    [NormedAddCommGroup F] [NormedSpace K F] (L : Submodule ℤ E)
+
+/--- The coimage of a `ZLattice` by a linear map. -/
+protected def ZLattice.comap (e : F →ₗ[K] E) := L.comap (e.restrictScalars ℤ)
```

Comment:
Could you please mention this definition in the module docstring at the top of the file.
And could you please expand this docstring to briefly clarify why you call this `comap` despite the "coimage" in the docstring. (I think there is a good reason :smile:)